server/models/Expense.js

```diff
@@ -244,6 +246,10 @@ export default function(Sequelize, DataTypes) {
     return this.getUser().then(user => user.paypalEmail || user.email);
   };
 
+  Expense.prototype.getFromCollective = function() {
+    return models.Collective.findByPk(this.FromCollectiveId);
```

Comment:
In https://github.com/opencollective/opencollective-api/pull/3231/files#diff-8f509dcd331d60429c7be6e52df71925R205 I've modeled the relation properly by adding:

```es6
  m.Expense.belongsTo(m.Collective, {
    foreignKey: 'FromCollectiveId',
    as: 'fromCollective',
  });
```

You can add that here if you want, with it you don't need to create the method manually sequelize will do it for you.